compliance/github/repository_branch_protection/repository_branch_protection.pt

```diff
@@ -21,12 +21,6 @@ parameter "param_orgs" do
   label "Organizations on GitHub.com to scan"
 end
 
-parameter "param_branches" do
```

Comment:
Why are we removing functionality.  since this is a customer public policy we shouldn't remove functionality.

Reply:
I added back the functionality.

Reply:
@cdwilhelm with my latest changes branches can be specified to scan like before, can you please have a look at the new changes when you have a minute?